.shippable.yml

```diff
@@ -80,7 +80,7 @@ build:
     - _make PLATFORM=imx-mx6ulevk
     - _make PLATFORM=imx-mx6ulevk CFG_NXP_CAAM=y CFG_CRYPTO_DRIVER=y
     - _make PLATFORM=imx-mx6ul9x9evk
-    - _make PLATFORM=imx-mx6ullevk
+    - _make PLATFORM=imx-mx6ullevk CFG_WITH_SOFTWARE_PRNG=n CFG_IMX_RNGB=y
```

Comment:
You may want to do this change in ```.azure-pipelines.yaml``` also.
Looks like ```.shippable.yml``` is about to be killed https://github.com/OP-TEE/optee_os/pull/4452